### PR TITLE
added gradient

### PR DESCRIPTION
**Pull request by swaager**
https://swaager.github.io/OpenCode-Collaborative/

### CODE REVIEW

**Comment by swaager** [comment]
@Anupam-dagar  , it won't look good if i will decrease the height of the progress bar , still u want i can change that. plzz review

File: json/game.json [hunk 1]
```diff
@@ -47,12 +47,15 @@
           "imageurl": "game/pong.png",
           "gamename" : "Ping-Pong-Game",
           "github" : "https://maanasvohra.github.io/2D-Ping-Pong-Game/"
+<<<<<<< HEAD
+=======
```
Comment (pradeepgangwar) [inline json/game.json]: Remove these lines.

**Comment by swaager** [comment]
@pradeepgangwar  done !!!

**Comment by aswinzz** [comment]
@swaager  can you sent a screenshot of the reduced one

**Comment by swaager** [comment]
oe
@aswinzz  !!!

**Comment by Anupam-dagar** [comment]
@swaager why the color is visible after some grey gap

**Comment by gauthamzz** [comment]
deepinscreenshot_20180123205335

**Comment by swaager** [comment]
https://swaager.github.io/OpenCode-Collaborative/
Hope this is done!!

**Comment by Anupam-dagar** [comment]
make the background transparent and change gradient to the one used in website

**Comment by swaager** [comment]
#438 
I have send new PR

**Pull request closed without merging**